packages/@azure/eventhubs/client/changelog.md

```diff
@@ -1,3 +1,16 @@
+### 2019-03-26 2.0.0
+
+#### Breaking Changes
+- If you have been using the `createFromAadTokenCredentials` function to create an instance of the `EventHubClient`, you will now need to use the [@azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) library instead of [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) library to create the credentials that are needed by the `createFromAadTokenCredentials` function.
+- If you have been passing a non string value in the `partitionKey` property on the message when sending it using the `EventHubClient`, an error will now be thrown. This property only supports string values.
+
+#### Bug fixes
+- A network connection lost error is now treated as retryable error. A new error with name `ConnectionLostError` is introduced for this scenario which you can see if you enable the logs.
```

Comment:
Can we have a snippet for how to show that in the logs?

Reply:
Added link to the readme on logs